buoy_course:
  ros__parameters:
    red_buoys_x: [2.0, 3.0, 5.0, 7.0, 14.0]
    red_buoys_y: [4.0, 7.0, 9.0, 9.7, 15.0]
    green_buoys_x: [3.5, 4.4, 6.0, 7.0, 15.0]
    green_buoys_y: [4.0, 6.35, 7.8, 8.0, 13.5]